src/Orckestra.Composer.Website/UI.Package/Sass/Lib/bootstrap/_nav.scss

```diff
@@ -32,7 +32,7 @@
 //
 
 .nav-tabs {
-  border-bottom: $nav-tabs-border-width solid $nav-tabs-border-color;
+  //border-bottom: $nav-tabs-border-width solid $nav-tabs-border-color;
 
```

Comment:
if not needed just remove

Reply:
Going to override it as this is a base bootstrap file